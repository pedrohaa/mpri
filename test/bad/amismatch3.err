Fatal error: exception Invalid_argument("List.map2")
