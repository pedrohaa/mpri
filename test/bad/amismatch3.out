Esquema: forall a_1_. { a_1_; list a_1_ } -> list a_1_
